src/controller.cpp

```diff
@@ -267,6 +276,9 @@ void Controller::activateInternal() {
       serverUnavailable();
       return;
     }
+    // NOTE: For platforms without multihop support, we cannot emulate multihop
+    // and use port 53 at the same time. If the user has selected both options
+    // then let's choose multihop.
```

Comment:
Should we maybe have a VPNCheckBoxAlert similar to the "vpn needs to be off" below the setting checkbox when multihop-mobile is active? Something like "this setting is ignored when multihop is active?"